crates/codegen/parser/runtime/src/parse_output.rs

```diff
@@ -18,4 +18,8 @@ impl ParseOutput {
     pub fn is_valid(&self) -> bool {
         return self.errors.is_empty();
     }
+
+    pub fn create_tree_cursor(&self) -> Cursor {
```

Comment:
If exposing the API is important for us now, I would add more documentation to the public cursor functions that the user might use

Reply:
Sure!